### ISSUE TITLE
LibrariesContainer fails to open in Multi-Module Compose App (v10.6.2)

### ISSUE THREAD

**Issue by GoldenSoju** [mikepenz/AboutLibraries#865]
## About this issue

- LibrariesContainer fails to open in Multi-Module Compose App due to error:

```
java.lang.IllegalStateException: Please provide the required library data via the available APIs.
Depending on the platform this can be done for example via `LibsBuilder().withJson()`.
For Android there exists an `LibsBuilder.withContext()`, automatically loading the `aboutlibraries.json` file from the `raw` resources folder.
When using compose or other parent modules, please check their corresponding APIs.
```                                                                                                    

I wasn't able to find any aboutlibraries.json file in the build folders. So I guess I have to adjust the gradle settings/setup (?), but I wasn't able to solve this on my own yet.

- How can the issue be reproduced / sample code

My app architecture is best compared to the Google **[Now In Android](https://github.com/android/nowinandroid)** app architecture. So when I add the dependencies for the about libraries plugin `aboutlibraries-plugin = {group = "com.mikepenz", name = "aboutlibraries-compose", version.ref = "aboutLibraries"}` with version `aboutLibraries = "10.6.2"` and try to open the `LibrariesContainer()` in a feature module. The app crashes and  I get the exception mentioned above.

## Details
- [ x ] Used library version
- [ ] Used support library version
- [ ] Used gradle build tools version
- [ ] Used tooling / Android Studio version
- [ ] Other used libraries, potential conflicting libraries

## Checklist

- [ x ] Searched for [similar issues](https://github.com/mikepenz/AboutLibraries/issues)
- [ x ] Checked out the [sample application](https://github.com/mikepenz/AboutLibraries/tree/develop/app)
- [ x ] Read the [README](https://github.com/mikepenz/AboutLibraries/blob/develop/README.md)
- [ x ] Checked out the [CHANGELOG](https://github.com/mikepenz/AboutLibraries/releases)
- [ x ] Read the [MIGRATION GUIDE](https://github.com/mikepenz/AboutLibraries/blob/develop/MIGRATION.md)

**Comment by mikepenz** [comment]
This looks to be a duplicate of: https://github.com/mikepenz/AboutLibraries/issues/858

**Comment by GoldenSoju** [comment]
@mikepenz 
Ah indeed. Seems like the same problem. So let me close this one.

**Comment by mikepenz** [comment]
Thank you. There is sadly no direct answer yet on the other to fix it right away. 

The assumption is that something in the gradle system breaks.

**Comment by GoldenSoju** [comment]
@mikepenz 

For the time being I switched to google's oss plugin as it was implemented in [nowinandroid #730](https://github.com/android/nowinandroid/pull/730). So far it works without problems in a multi module app.
